docker/docker-compose.dev.yml

```diff
@@ -28,12 +28,13 @@ services:
   immich-machine-learning:
     container_name: immich_machine_learning
     image: immich-machine-learning-dev:latest
+    shm_size: 256MiB
```

Comment:
Ray uses shared memory to pass data between processes quickly. If this is omitted, then `/tmp` will be used instead of `/dev/shm`, which is somewhat slower.